### PR TITLE
Discuss sites that users pay for with data.

### LINKED ISSUE #189: Write (or remove) section on "pay for use in data"
An in-line issue to be resolved:

> ISSUE
> Some services have the user pay for their use in data. These services aren't necessarily retaliating by denying their services to users who refuse to pay with data, but the details are more complex than we've had time to write.

https://w3ctag.github.io/privacy-principles/#non-retaliation

### PR DESCRIPTION
**Pull request by jyasskin**
Fixes #189.

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/436.html" title="Last updated on Oct 23, 2024, 4:42 PM UTC (48a21c9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/436/4337d7b...jyasskin:48a21c9.html" title="Last updated on Oct 23, 2024, 4:42 PM UTC (48a21c9)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2046,6 +2046,10 @@
 unrelated feature, by decreasing the quality of the service, or by trying to cajole, badger, or
 trick the [=person=] into opting back into the [=processing=].
 
+Note that because data is valuable, some sites allow users to pay for use of the site with data. If
+a user of this sort of site doesn't [=consent=] to exchange their data for use of the site, it's not
+retaliation to prevent them from using the site.
```
Comment (martinthomson) [inline index.html:2051]: Is there anything that then needs to be said about the nature of such deals?  https://en.wikipedia.org/wiki/Hobson's_choice perhaps?

Comment (dmarti) [reply index.html:2051]: We can't make the blanket statement that denying some web resource to users who give some data consent is never retaliation. The "pay for the use of a site with data" covers a lot of different situations, from "fill in your email address to read a white paper" to "consent to surveillance on all your devices in order to get to your child's school announcements."

There are "controlled circulation" business models in publishing where a legit exchange of lead gen data for content takes place, but also cases where the power relationship between a site and user does not allow for actual consent.

Maybe better to say "some sites offer a free and fair exchange of appropriate information for original content they create or pay for the creation of, and such an exchange is not retaliation."

**Review by dmarti — CHANGES_REQUESTED** [review]
We can't make the blanket statement that denying some web resource to users who choose not to give some consent is never retaliation. The "pay for the use of a site with data" covers a lot of different situations, from "fill in your email address to read a white paper" to "consent to surveillance on all your devices in order to get to your child's school announcements."

There are "controlled circulation" business models in publishing where a legit exchange of lead gen data for content takes place, but also cases where the power relationship between a site and user does not allow for actual consent.

Maybe better to say "some sites offer an informed, free and fair exchange of appropriate user information for original content they create or pay for the creation of, and denying access to users who choose not to participate in such an exchange is not retaliation."

**Comment by jyasskin** [comment]
I'll replace the PR with @dmarti's suggestion.

**Comment by jyasskin** [comment]
How's this?

**Review by dmarti — APPROVED** [review]
This looks good to me, thank you.

**Comment by jyasskin** [comment]
This was reverted by #451.

**Pull request merged**